### ISSUE TITLE
Better accessibility for qooxdoo apps (Implement ARIA)

### ISSUE THREAD

**Issue by cboulanger** [qooxdoo/qooxdoo#9685]
Currently, qooxdoo lacks any special support for screenreaders and other devices that help, for example, sight-impaired users to navigate and use qooxdoo applications. 
We need to implement [ARIA roles, properties and states](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/WAI-ARIA_basics) in the HTML generated by the qooxdoo runtime.

**Comment by hkollmann** [comment]
Done
